src/components/scenes/GettingStartedScene.tsx

```diff
@@ -100,19 +100,16 @@ export const GettingStartedScene = (props: Props) => {
 
   const sections: SectionData[] = sectionsVariantMap[landingType] ?? sectionsVariantMap.default
 
-  // An extra index is added to account for the extra initial usp slide OR to
   // allow the SwipeOffsetDetector extra room for the user to swipe beyond to
+  // An extra index is added to account for the extra initial usp slide OR to
```

Comment:
This was accidentally moved.

Reply:
I don't see anything moved. If you meant removing a line of the comment, that was intentional. `swipeLastUsp` is no longer under AB test